extensions/classifiers/LDAStringClassifier/LDAStringClassifier.py

```diff
@@ -627,9 +690,7 @@ def validate(self, classifier_data):
             '_w_dp',
             '_l_dp',
             '_c_dl',
-            '_c_lw'
-        ]
-        list_of_list_bool_properties = [
+            '_c_lw',
             '_b_dl'
```

Comment:
That's not what I meant. Here, you've removed a bool property and turned it into an int property. If the data is expected to be booleans, the validation function should check that it is actually booleans.